Common/include/CConfig.hpp

```diff
@@ -865,7 +865,7 @@ class CConfig {
   nMu_Temperature_Ref,           /*!< \brief Number of species reference temperature for Sutherland model. */
   nMu_S,                         /*!< \brief Number of species reference S for Sutherland model. */
   nThermal_Conductivity_Constant,/*!< \brief Number of species constant thermal conductivity. */
-  nPrandtl_Lam,                  /*!< \brief Number of species laminar Prandtl number. */
```

Comment:
The original description was correct, for species transport we can have a Prandtl number for each of the species

Reply:
Yeah I do not recall changing that but for sure it was wrong in my branch. Corrected, thanks!